src/chatbot/chatbot.component.ts

```diff
@@ -51,6 +55,7 @@ export class ChatbotComponent implements OnInit {
   public isLoading = false;
   public loadingIndex: number;
   public isOpen = false;
+  public disableChatbotByAction = '';

```

Comment:
"currentAction"? It implies an on-going action. Disabling the UI is a side-effect of there being such an on-going action. In the future there may be other side-effects.

Reply:
Fixed.